### ISSUE TITLE
Issue with whitelist

### ISSUE THREAD

**Issue by bertho-zero** [feathersjs-ecosystem/feathers-knex#215]
If in the options I add `whitelist: [ '$search' ]` then my service does not recognize `$like` etc, I have to add them all.

```
// does not work
[
  '$search'
]
```

```
// does work
[
  '$like',
  '$notlike',
  '$ilike',
  '$and',
  '$search'
]
```

I do not know if this is the desired behavior?

**Comment by daffl** [comment]
It always adds them to the list in https://github.com/feathersjs-ecosystem/feathers-knex/blob/master/lib/index.js#L44 - do you have the latest version (v6.0.1)?

**Comment by bertho-zero** [comment]
I have the latest version.

https://github.com/feathersjs-ecosystem/feathers-knex/blob/6bc2d9da0b8628cc10c9c657c085aba52338ea55/lib/index.js#L40-L45

If `options.whitelist` exists then the concat is overwritten.

**Comment by daffl** [comment]
🤦‍♂ Well, that makes sense. Fixed in 6.0.2
